supernode/cmd/app.go

```diff
@@ -132,20 +131,22 @@ func runApp(ctx context.Context, config *configs.Config) error {
 
 	// p2p service (currently using kademlia)
 	config.P2P.SetWorkDir(config.WorkDir)
-	p2p := p2p.New(config.P2P)
+	// p2p := p2p.New(config.P2P)
```

Comment:
why comment it?

Reply:
@pjenys0 there are some thing wrong with p2p client so I commented it because it print a lot of logs. Revert the comment now.